cnxauthoring/storage/postgresql.py

```diff
@@ -366,3 +366,7 @@ def search(self, limits, type_=Document, submitter_id=None):
             for item in res:
                 yield self._reassemble_model_from_document_entry(**item)
         raise StopIteration
+
+    def restart(self):
+        """Restart the interface"""
+        self.conn = psycopg2.connect(self.conn.dsn)
```

Comment:
Any process that has access to the ram of this process to read the password probably can get to
the database anyway. The theoretical problem would be something heartbleed like, that can read random memory. But the db is configured to only allow connections from the servers themselves anyway, so the password is of little extra use.